### PR TITLE
Remove magic variables

### PR DESCRIPTION
**Pull request by liampan**

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala [hunk 1]
```diff
@@ -159,7 +160,7 @@ class SubscriptionControllerSpec
           response.processingDate mustBe subscriptionUpdateResponse.processingDate
           response.nrSubmissionId mustBe nrSubmissionId
           verify(mockNonRepudiationService).submitNonRepudiation(
-            ArgumentMatchers.eq("ppt-subscription"),
+            ArgumentMatchers.eq(NotableEvent.PptSubscription),
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala:163]: These tests should still assert that actual string as that is what the NRS is expecting (as we found out when we changed it!)

Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala:163]: So these tests dont and shouldnt cos they now have to pass the object which can only be one of two, the tests on the service its self check for the actual string where it converts the object to a sting by calling .id

Comment (liampan) [reply test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala:163]: which one doesnt, i tell a lie i will update that one...

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/services/nonRepudiation/NonRepudiationServiceSpec.scala [hunk 2]
```diff
@@ -113,7 +113,7 @@ class NonRepudiationServiceSpec
       await(res) mustBe NonRepudiationSubmissionAccepted("testSubmissionId")
 
       val testEncodedPayload = Base64.getEncoder.encodeToString(testPayloadString.getBytes(StandardCharsets.UTF_8))
-      val expectedMetadata = NonRepudiationMetadata(businessId = "ppt", notableEvent = NotableEvent.PptReturn.id,
+      val expectedMetadata = NonRepudiationMetadata(businessId = "ppt", notableEvent = "ppt-return",
```
Comment (liampan) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/services/nonRepudiation/NonRepudiationServiceSpec.scala:116]: here

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala (hunk 1 above)
Comment (lmcewanhmrc) [reply test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/controllers/SubscriptionControllerSpec.scala:163]: ok that makes sense

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/services/nonRepudiation/NonRepudiationServiceSpec.scala (hunk 2 above)
[the remark this answers is not in the document]
Comment (lmcewanhmrc) [reply test/uk/gov/hmrc/plasticpackagingtaxreturns/services/nonRepudiation/NonRepudiationServiceSpec.scala:116]: Do we check for the other notable event string somewhere already? Thanks

**Review by lmcewanhmrc — APPROVED** [review]

**Pull request merged**